tests/core/pyspec/eth2spec/test/eip7928/test_block_access_list.py

```diff
@@ -0,0 +1,38 @@
+"""
+Tests for EIP-7928 block access list field.
+"""
+
+from eth2spec.test.context import (
+    spec_state_test,
+    with_phases,
+)
+from eth2spec.test.helpers.constants import EIP7928
+from eth2spec.test.helpers.execution_payload import (
+    build_empty_execution_payload,
+    compute_el_block_hash,
+)
+from eth2spec.test.helpers.state import next_slot
+
+
+@with_phases([EIP7928])
```

Comment:
I believe this should be `@with_eip7928_and_later` (will need to create the decorator).